src/domain/hanke/hankeUsers/hankeUsersApi.ts

```diff
@@ -17,8 +17,8 @@ export async function updateHankeUsers({
   return data;
 }
 
-// Get user id and rights of the signed in user
-export async function getSignedInUser(hankeTunnus?: string): Promise<SignedInUser> {
+// Get user id and rights of the signed in user for a hanke
+export async function getSignedInUserForHanke(hankeTunnus?: string): Promise<SignedInUser> {
```

Comment:
Wouldn't a null hanketunnus lead to an error here? Is it a possible scenario?

Reply:
It should not be a possible scenario as the query is not enabled if hanketunnus is null.